app/views/document_sets/_edit.html.slim

```diff
@@ -13,7 +13,7 @@
       tr
         th.hidden= t('.url')
         td 
-          p.nomargin.settings ==t('.current_url_for_this_work', current_url: "<b>www.#{Rails.application.routes.default_url_options[:host]}/#{@document_set.owner.slug}/#{@document_set.slug}</b>")
+          p.nomargin.settings ==t('.current_url_for_this_work', current_url: "<b>#{Rails.application.routes.default_url_options[:host]}/#{@document_set.owner.slug}/#{@document_set.slug}</b>")
```

Comment:
I believe we can use the url_helper for this too. That way no need to conditionally append `www.`

Reply:
I couldn't find the right helper. Which one do you recommend?

Reply:
I think this is `collection_path(@document_set.owner.slug, @document_set.slug)` if I am not mistaken.

Reply:
If we need the full url here then collection_url should work.